tests/e2e/after.test.ts

```diff
@@ -25,7 +28,9 @@ test('next/after callback is executed and finishes', async ({ page, after }) =>
 
   expect(pageInfo2.timestamp, 'Check page should be cached').toBe(pageInfo1.timestamp)
 
-  await page.goto(`${after.url}/after/trigger`)
+  const response = await page.goto(`${after.url}/after/trigger`)
+
+  expect(response?.status(), 'Trigger should return 200').toBe(200)
```

Comment:
In `main` this results in 50x status (due to no longer existing export) - the test failed which is good, but assertion errors didn't show much details other than things not working

This added assertion is just to have more context about potential future failures